prov/tcp/src/tcpx_conn_mgr.c

```diff
@@ -260,6 +260,15 @@ static int send_conn_req(struct poll_fd_mgr *poll_mgr,
 	return ret;
 }
 
+static int tcpx_ep_msg_xfer_enable(struct tcpx_ep *ep)
+{
+	fastlock_acquire(&ep->cm_state_lock);
+	ep->cm_state = TCPX_EP_CONN_ACTIVE;
```

Comment:
Do we need to check that we're in the correct state here before changing it?  E.g. could the ep be disconnected for some reason prior to the user attempting to activate it?